packages/gatsby/src/commands/develop.ts

```diff
@@ -106,20 +112,25 @@ module.exports = async (program: IProgram): Promise<void> => {
   // Run the actual develop server on a random port, and the proxy on the program port
   // which users will access
   const proxyPort = program.port
-  const [statusServerPort, developPort] = await Promise.all([
+  const [statusServerPort, developPort, debugPort] = await Promise.all([
     getRandomPort(),
     getRandomPort(),
+    program.inspectBrk ? getRandomPort() : Promise.resolve(null),
```

Comment:
When using `node --inspect(-brk)` port by default is 9229 (not random). 

It would also be great if we could follow `node --inspect-brk` semantics - which allow us to specify port `--inspect-brk=9230` or skip specyfying port and then fallback to default

Reply:
@pieh have you seen my motivation for this in the linked issue. There's some complication around doing what you suggested. I think this could only be acheived if we forgo trying to debug the "parent process" and focus only on the "child process"

Reply:
Thanks for the nudge to go look into the issue discussion! I'll take this discussion there ;)